packages/core/plugins/core-plugins-server-internal/src/plugins_system.ts

```diff
@@ -250,10 +250,10 @@ export class PluginsSystem<T extends PluginType> {
         try {
           const resultMaybe = await withTimeout({
             promise: plugin.stop(),
-            timeoutMs: 30 * Sec,
+            timeoutMs: 15 * Sec,
```

Comment:
nitnit: for posterity, I wonder if it's worth capturing a small bit of context on this number in the code. Maybe something like:

```
Setting this value higher decreases the likelihood that all plugins' .stop() function can execute if there is a plugin that takes a long time stop.
```

Can go without too though.